codec-http2/src/main/java/io/netty/handler/codec/http2/Http2ConnectionHandler.java

```diff
@@ -53,11 +57,14 @@
  */
 public class Http2ConnectionHandler extends ByteToMessageDecoder implements Http2LifecycleManager {
     private static final InternalLogger logger = InternalLoggerFactory.getInstance(Http2ConnectionHandler.class);
+    private static final int DEFAULT_GRACEFUL_SHUTDOWN_TIMEOUT_MILLIS = 15 * 60 * 1000; // 15 minutes
```

Comment:
@Scottmitch Yeah now that I think about it, I think this makes sense.  +1 to 30 seconds.